shell/client/admin/user-details-client.js

```diff
@@ -230,4 +300,67 @@ Template.newAdminUserDetails.events({
       isAdmin: false,
     });
   },
+
+  "click [name=\"delete-account\"]"(evt, instance) {
```

Comment:
Any particular reason you are using `name` to distinguish buttons here, rather than class as we usually do?